lepotato/debimage-lepotato.yaml

```diff
@@ -0,0 +1,68 @@
+{{- $architecture := or .architecture "arm64" -}}
+{{- $suite := or .suite "sid" -}}
```

Comment:
Defaulting to `sid` seems like it'll likely sooner or later end up with the recipe breaking. If buster isn't new enough, then hopefully using `bullseye` would work and be more stable.